engine/execution/computation/computer/computer_test.go

```diff
@@ -1203,7 +1203,7 @@ func Test_ExecutingSystemCollection(t *testing.T) {
 	noopCollector := metrics.NewNoopCollector()
 
 	expectedNumberOfEvents := 3
-	expectedEventSize := 1435
+	expectedEventSize := 1436
```

Comment:
any specific reason behind this increase?

Reply:
This is connected to the random beacon. If the state commitment changes, the random seed changes, which can influence the length of one event that is emited.